android/src/main/java/vn/luongvo/kmm/survey/android/MainActivity.kt

```diff
@@ -0,0 +1,87 @@
+package vn.luongvo.kmm.survey.android
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import vn.luongvo.kmm.survey.Greeting
+
+@Composable
+fun MyApplicationTheme(
```

Comment:
Please create a new file for this

Reply:
Same, next task https://github.com/luongvo/kmm-survey/issues/2